ruma-serde/src/boolean.rs

```diff
@@ -0,0 +1,81 @@
+//! De-/serialization functions for casting boolean values to int and vise versa.
+//! Delegates to `js_int::UInt` to ensure integer size is within bounds.
```

Comment:
The bounds here are really `[0, 1]` though, right? I think this could use really use any integer type, unless the idea in the synapse admin API is actually to allow any non-`0` value and then it'd probably be sensible to use `u64` or `u128` instead.

🤷🏼